Libs/GObject-2.0/Classes/Object.cs

```diff
@@ -97,8 +99,11 @@ protected internal void OnPropertyChanged([CallerMemberName] string? propertyNam
 
         public virtual void Dispose()
         {
+            Debug.WriteLine($"Disposing Object: Address {_handle.Handle}, Type {GetType()}.");
             _signalRegistry.Dispose();
             _handle.Dispose();
         }
+
+        ~Object() => Dispose();
```

Comment:
This should not be needed. The GC should automatically garbage collect our instances. I wonder how it works for the signalRegistry in the `Dispose` method. Needs to be evaluated.

Reply:
The signal registry is a local object of the instance itself. It just gets garbage collected.